controllers/gcpcluster.go

```diff
@@ -137,10 +162,37 @@ func (r *GCPClusterReconciler) reconcileDelete(ctx context.Context, gcpCluster *
 	return ctrl.Result{}, nil
 }
 
-func (r *GCPClusterReconciler) createBastionFirewallRule(ctx context.Context, logger logr.Logger, gcpCluster *capg.GCPCluster) error {
+func (r *GCPClusterReconciler) applyAPISecurityPolicy(ctx context.Context, logger logr.Logger, gcpCluster *capg.GCPCluster) error {
+	policyName := getAPISecurityPolicyName(gcpCluster.Name)
+	sourceIPRanges, err := getIPRangesFromAnnotation(logger, gcpCluster, AnnotationAPIAllowListSubnets)
```

Comment:
IMHO the annotation should only contain customer IPs, our IP of VPN and MC outgoing Ips should be added by some other way so we do not confuse customers and also we don't allow customer to remove our IPs from the list